packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -71,57 +69,65 @@ export const PaymentConfirmation = ({
   const downloadUrl: URL = new URL(productUrl);
   downloadUrl.searchParams.append('email', email);
 
+  // TW classes
+  const subheadingClasses = 'text-grey-400 max-w-sm';
+  const h2classes = 'text-xl font-normal mx-0 mt-6 mb-3';
+  const h3classes = 'text-sm font-semibold';
+  const bottomRowClasses = 'flex justify-between items-center text-grey-400';
+
   return (
     <>
       <SubscriptionTitle screenType="success" />
       <section
-        className={`container card payment-confirmation ${className}`}
+        className="container payment-confirmation"
         data-testid="payment-confirmation"
       >
-        <header>
+        <header
+          className={`flex flex-col justify-center items-center row-divider-grey-200 text-center pb-8 mt-5 desktop:mt-2`}
```

Comment:
nit: Since we aren't using any expressions or nothing, should we use double quotes instead of template literal strings?
```suggestion
          className="flex flex-col justify-center items-center row-divider-grey-200 text-center pb-8 mt-5 desktop:mt-2"
```

Reply:
Thank you! I updated previous code that used a `const`. Excellent catch. Fixed!